File: C-Programming-Practise/Programming-Tutorial-For-Beginners-Learn-C/for_loop.c
#include <stdio.h>
int main()
{
    int num=1, input;
    scanf("%d", &input);
    printf("how many times you want the loop to run?\n");

    for (num; num <= input; num++)
    {
        printf("%d\n", num);
    }
    
    
    return 0;
}

File: Java-Programming-Practise/Java-tutorial-in-hindi/For_Loop.java
package com.java;

import java.util.Scanner;

public class For_Loop {
    public static void main(String[] args) {
        int input, i=1;

        Scanner scanner = new Scanner(System.in);

        System.out.println("How many times you want loop to run?");

        input = scanner.nextInt();

        for (; input >= i; i++){
            System.out.println(i);
        }
    }
}
